packages/latte-phpstan-compiler/composer.json

```diff
@@ -0,0 +1,58 @@
+{
+    "name": "symplify/latte-phpstan-compiler",
+    "description": "Compile latte template to PHPStan-compatible PHP code to allow its static analysis",
+    "license": "MIT",
+    "require": {
+        "php": ">=8.0",
+        "nette/utils": "^3.2"
```

Comment:
latte/latte isn't dependency? compiler is there and also Defaults class etc

Reply:
Yes, this is WIP. I'm still commiting :)